### PR TITLE
donut and progress charts

### PR DESCRIPTION
**Pull request by mkmarek**

### CODE REVIEW

File: src/components/atoms/ProgressBar/ProgressBar.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    max: PropTypes.number.isRequired,
+    barColor: PropTypes.string,
+    disabled: PropTypes.bool
+}
+
+ProgressBar.defaultProps = {
+    barColor: 'green'
+}
+
+export default ProgressBar
```
Comment (alexyork) [inline src/components/atoms/ProgressBar/ProgressBar.jsx]: All JS lines should end with semi-colons. I know this is optional in JS, but we have simply too much code by now to go for the no semi-colons, and now the best thing to do is be consistent. Thanks!

File: src/components/atoms/Spinner/Spinner.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Spinner.propTypes = {
+    className: PropTypes.string,
+    type: PropTypes.string
+}
+
+Spinner.defaultProps = {
+    type: 'lg'
+}
+
+export default Spinner
```
Comment (alexyork) [inline src/components/atoms/Spinner/Spinner.jsx]: Semi-colons.

File: src/components/molecules/DonutChart/DonutChart.jsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    loadingCaption: "Henter data...",
+    valueCaption: "gjenstår",
+    thresholds: [
+        { to: 0.2, type: 'red' },
+        { to: 0.6, type: 'orange' },
+        { to: 1, type: 'green' }
+    ]
+}
+
+export default DonutChart
```
Comment (alexyork) [inline src/components/molecules/DonutChart/DonutChart.jsx]: Semi-colon.

File: src/components/molecules/ProgressChart/ProgressChart.jsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    loadingCaption: "Henter data...",
+    valueCaption: "gjenstår",
+    thresholds: [
+        { to: 0.2, type: 'red' },
+        { to: 0.6, type: 'orange' },
+        { to: 1, type: 'green' }
+    ]
+}
+
+export default ProgressChart
```
Comment (alexyork) [inline src/components/molecules/ProgressChart/ProgressChart.jsx]: Semi-colons.

File: src/components/molecules/ProgressChart/ProgressChart.pcss [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        max-height: 20px;
+
+        top: 0;
+        padding: 0 1em;
+        display: flex;
+        position: relative;
+
+        text-align: center;
+        line-height: 1.3rem;
+        white-space: nowrap;
```
Comment (alexyork) [inline src/components/molecules/ProgressChart/ProgressChart.pcss]: Is there a reason behind this CSS grouping...?

File: src/components/colors.pcss [hunk 6]
```diff
@@ -39,4 +41,5 @@
     --red: #FF3264;
     --dark-red: #FF2364;
     --light-red: #FF6464;
+    --venetian-red: #D0021B;
```
Comment (alexyork) [inline src/components/colors.pcss]: I will double-check what we do when we are presented with new colours. The ones we already have come from the Telia Company brand, and we should try and use them as much as possible. I will get back to you :-)

File: src/components/molecules/ProgressChart/ProgressChart.pcss (hunk 5 above)
Comment (mkmarek) [reply src/components/molecules/ProgressChart/ProgressChart.pcss]: After I had a second look it was pretty nonsensical. However I tried to separate styles that define different aspects of a block. Now I separated it into

- stuff that influences the element display properties (position, display)
- stuff that defines area around the content (padding, margin, top, bottom, ...)
- stuff that defines text (line-height, font-size, text-align, ...)

Not sure if you're using different approaches, or none at all but I can adjust.

**Pull request merged**